Products/Zuul/routers/device.py

```diff
@@ -398,6 +398,7 @@ def getDeviceUuidsByName(self, query="", start=0, limit=25, page=1, uuid=None):
         If uuid is set, ensures that it is included in the returned list.
         """
         facade = self._getFacade()
+        query = "*" if not query else query
```

Comment:
Can we just put `query="*"` as an argument of a method

`def getDeviceUuidsByName(self, query="*", start=0, limit=25, page=1, uuid=None):`